lua/weapons/gmod_tool/stools/hoverball_spawner.lua

```diff
@@ -27,6 +29,8 @@ function TOOL:LeftClick( trace )
         if (IsValid(trace.Entity)) then
             local weld = constraint.Weld( entity, trace.Entity, 0, trace.PhysicsBone, 0, true , false )
         end
+            self:GetOwner():AddCount( "Marums Hovers", entity )
```

Comment:
why is this here? this function is used for sbox entity limits not cleanup functions

Reply:
/shrug
Ech i was looking at the original gmod hoverball code, and i just basically replicated it.